src/js/helpers/covid19Helper.js

```diff
@@ -7,6 +7,7 @@ import { snakeCase } from 'lodash';
 import { defCodes } from 'dataMapping/covid19/covid19';
 import { componentByCovid19Section } from 'containers/covid19/helpers/covid19';
 import { scrollToY } from 'helpers/scrollToHelper';
+import { apiRequest } from './apiRequest';
```

Comment:
So far we've been putting API requests in `helpers/disasterHelper` and using this file for helper functions